js/djtilesource.js

```diff
@@ -1,92 +1,134 @@
+/*jslint browser: true*/
+/*global OpenSeadragon, Drupal*/
+/**
+ * @file
+ * Defines the source for exposing Djatoka to OpenSeadragon.
+ */
 (function($) {
+  'use strict';
+
+  // Remove IIFTileSource, as it assumes Djatoka responses belong to it for some reason.
+  $.IIIFTileSource = undefined;
```

Comment:
'For some reason' in this cause is how Openseadragon determines which tilesource to instantiate. I wasn't keen on including all of this in a comment in the source as it's implementation details from Openseadragon that may change with later updates to Openseadragon and be incorrect or misleading to later readers. 

This is the skinny on it: 

https://openseadragon.github.io/docs/openseadragon.js.html 

1. The function `getTileSourceImplementation` is responsible for instantiating the tile source used by Openseadragon.
2. In particular cases it will call `$.TileSource.determineType` to resolve which type a given configuration is.
3. This function loops through the properties in the Openseadragon object, or namespace as that's more so the intention in this case. Looking for ones with that end in `TileSource`. For each it finds it calls the function `supports` on each. If this function returns true, it will use instantiate this 'type' of tile source.

One of the problems with Openseadragons approach here is what to do when multiple implementations of `$.TileSource` return `true`? `IIIFTileSource` simply checks the url for the tile source looks like IIIF url if so, then returns `true` so our `DjatokaTileSource` is not considered, even though in this case we want it to use the `DjatokaTileSource` explicitly.